temba/channels/tests.py

```diff
@@ -4696,7 +4733,7 @@ def test_received(self):
 
         # load our message
         msg = Msg.objects.get()
-        self.assertEquals('+977788123123', msg.contact.get_urn(TEL_SCHEME).path)
+        self.assertEquals('977788123123', msg.contact.get_urn(TEL_SCHEME).path)
```

Comment:
This is due to the phonenumbers library updating which is now more accurate in rejecting nepalese numbers. This is not the correct fix, if you merge the master branch into yours, you'll get the proper fix for this.